### PR TITLE
feat: update apt/yum repos

### PR DESCRIPTION
**Pull request by bdossantos**
Hi,

I've updated yum/apt repository location, according documentation.

This also permit to get latest sensu version, v0.29

https://sensuapp.org/docs/0.29/platforms/sensu-on-ubuntu-debian.html#install-sensu-core-repository

Next step: ensure apt/yum https transport package is installed then use the HTTPS endpoint by default.

Thank you

### CODE REVIEW

**Comment by bdossantos** [comment]
Oups, broken specs. I'll try to fix it.

**Comment by jaxxstorm** [comment]
Any luck here?

**Comment by ghoneycutt** [comment]
Looks good, though tests are failing. Think it is related to http/https change.

File: manifests/init.pp [hunk 1]
```diff
@@ -360,7 +360,7 @@
   $repo                           = 'main',
   $repo_source                    = undef,
   $repo_key_id                    = 'EE15CFF6AB6E4E290FDAB681A20F259AEB9C94BB',
-  $repo_key_source                = 'http://repositories.sensuapp.org/apt/pubkey.gpg',
+  $repo_key_source                = 'https://sensu.global.ssl.fastly.net/apt/pubkey.gpg',
```
Comment (ghoneycutt) [inline manifests/init.pp]: We need to keep this http for now to ensure compatibility.

Comment (bdossantos) [inline manifests/init.pp]: Hi, sorry for the late reply, I'll fixup tomorrow.

Comment (cwjohnston) [inline manifests/init.pp]: @ghoneycutt https with `sensu.global.ssl.fastly.net` is OK. 

We need to be careful with https on `repositories.sensuapp.org`, as the cert being provided on that CDN map uses SNI (subjectAltName, not supported by all package managers) and requires TLS 1.2 (not supported by all openssl versions)

**Comment by bdossantos** [comment]
@ghoneycutt @cwjohnston done

**Comment by ghoneycutt** [comment]
Rebasing and testing

**Comment by ghoneycutt** [comment]
Thank @bdossantos for your contributions! Much appreciated.

**Comment by cwjohnston** [comment]
I believe the URL scheme needs to change from http to https if we want to realize a benefit from this.

**Comment by ghoneycutt** [comment]
This gets us on the correct hosts. I'll change the protocol next.

**Comment by ghoneycutt** [comment]
Rebased in #696

**Pull request closed without merging**